### DISCUSSION TITLE
Designing User Experience

### DISCUSSION THREAD

**Discussion by shane-andrus** [Green-Canyon-Computer-Science/Green-Canyon-App#1] [Team Posts]
A fantastic tool we can use to help design an effective user interface is [Figma](https://www.figma.com/). This is a tool that we can use to design what we want before we actually start coding.

**Comment by CompputerMonkey** [comment]
Cool! We should get some more people on board and talk about our goal and ideas.

**Comment by InvertedOwl** [comment]
Yea I imagine most of us arent UIX pros haha

**Comment by shane-andrus** [comment]
Very true. I was thinking that we could get some help with this.
